xml/docu_styleguide.language.xml

```diff
@@ -80,58 +80,13 @@
   <title>Capitalization and Title Style</title>
   <para>
    In all running text, use sentence-style capitalization. That is, only
-   capitalize the first word of a sentence and proper names.
+   capitalize the first letter and proper names.
```

Comment:
TBH I preferred the older formulation.

Reply:
Well, I figured you don't want to confuse anyone by not explicitly saying it's the letter. The older variant could easily mean the whole word should be capitalised.

Reply:
If we want to be precise, "the first letter of the first word".

Reply:
What @sknorr said.

"Capitalize only the first letter of the first word and of proper nouns" is more specific. Every word has a first letter, after all.